howso/utilities/installation_verification.py

```diff
@@ -273,6 +275,12 @@ def run_checks(self) -> int:  # noqa: C901
         start_time = datetime.now()
 
         try:
+            versions = get_versions()
```

Comment:
Can you wrap a try/except around this? If for some reason an exception is raised during version check, I'd hate for us to miss out on other potentially useful diagnostic information.